docs/usage/getting-started/private-packages.md

```diff
@@ -360,7 +360,8 @@ The end-result looks like this:
 
 #### Yarn 2+
 
-Renovate doesn't support reading `npmRegistries` and `npmScopes` from `.yarnrc.yml`, so `hostRules` (or `npmToken`) and `npmrc` should be configured like above.
+Renovate doesn't support reading `npmRegistries` and `npmScopes` from `.yarnrc.yml`, so `hostRules` (or `npmToken`) and `npmrc` should be configured like above. The `${NPM_TOKEN}` substitution that takes place for `.npmrc` will also not occur for `.yarnrc.yml`.
```

Comment:
```suggestion
Renovate doesn't support reading `npmRegistries` and `npmScopes` from `.yarnrc.yml`, so `hostRules` (or `npmToken`) and `npmrc` should be configured like above.
The `${NPM_TOKEN}` substitution that takes place for `.npmrc` will also not occur for `.yarnrc.yml`.
```

Reply:
This text is hard to read and understand. How about something like this?

> #### Yarn 2+
>
> Introductory sentence that explains the default Yarn 2+ configuration.
> If we recommend one method of Yarn use above another, say so here as well.
> I suppose the default Yarn 2+ config works well with Renovate?
> 
> ##### When you use a `.yarnrc.yml` file
>
> Renovate can _not_ read these these config options from `.yarnrc.yml` files:
>
> - `npmRegistries`
> - `npmScopes`
> 
> When you use a `.yarnrc.yml` file Renovate will _not_ substitute the `${NPM_TOKEN}`, like it does when you use a `.npmrc` file.
> 
> This all means you must configure your `hostRules` (or `npmToken`) and `npmrc` like ...insert proper reference to correct config here...